docs/user-guide/release-notes/2.16.x.rst

```diff
@@ -5,14 +5,31 @@ Pulp 2.16 Release Notes
 Pulp 2.16.2
 ===========
 
-This is a security fix for CVE-2018-1090 which leaks security info via the `last_override_config` on
-importers and distributors serialized via the Rest API. See `issue 3521 <https://github.com/pulp/pulp/pull/3513>`_
-for more information. With this release, that field no longer reports the information saved in the
-database. Instead it will always show as::
+This release contains a security fix for CVE-2018-1090 which leaks security info via the
+`last_override_config` on importers and distributors serialized via the Rest API. See `issue 3521
+<https://github.com/pulp/pulp/pull/3513>`_ for more information. With this release, that field no
+longer reports the information saved in the database. Instead it will always show as::
 
     'last_override_config': {}
 
 
+This release also contains some bug fixes for issues that were encountered while trying to run Pulp
+in a FIPS enabled environment. 
```

Comment:
I think we ought to include a list of issues:
https://pulp.plan.io/issues/3768 (this issue should be updated based on the associated revisions)
https://pulp.plan.io/issues/3646
https://pulp.plan.io/issues/3676

If you prefer, you can merge as is, and ill update in my PR.